### PR TITLE
docs: add logos and credits

### PR DESCRIPTION
**Pull request by MRigal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
+[![Technologiestiftung Berlin](https://img.shields.io/badge/Built%20with%20%E2%9D%A4%EF%B8%8F-at%20Technologiestiftung%20Berlin-blue)](https://www.technologiestiftung-berlin.de)
+
 # Innovationsdaten
 
-Daten aus 10 Jahre Innovationserhebung ab Innovationserhebung 2013
+Daten aus 10 Jahre Innovationserhebung seit 2012
```
Comment (MRigal) [inline README.md:5]: Corrected the information, but this should be in English. @marcoka2000 some suggestion about a slightly longer description?

File: README.md [hunk 2]
```diff
@@ -110,3 +113,30 @@ These **environment variables** should be set:
 | FONTS_URL      | <secret-URL> |
 | PORT           | 8000         |
 | PYTHON_VERSION | 3.11.4       |
+
+
+## Content Licencing
+
+Texts and content available as [CC BY](https://creativecommons.org/licenses/by/3.0/de/).
```
Comment (MRigal) [inline README.md:120]: @marcoka2000 is this true? Was it agreed/validated by SenWiEnBe and CH?

Comment (marcoka2000) [reply README.md:120]: Will mail CH about that, he probably knows more.

File: README.md (hunk 1 above)
Comment (marcoka2000) [reply README.md:5]: Will come up with a suggestion later today 👨🏻‍💻

Comment (MRigal) [reply README.md]: Looks good to me, thanks!

File: README.md (hunk 2 above)
Comment (marcoka2000) [reply README.md]: Ah, perfect - just wanted to do the same.
Confirmation just came in.

**Pull request merged**